pkg/components/testutils/componentversion.go

```diff
@@ -0,0 +1,120 @@
+package testutils
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+
+	cdv2 "github.com/gardener/component-spec/bindings-go/apis/v2"
+
+	lsv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
+	"github.com/gardener/landscaper/pkg/components/model"
+	"github.com/gardener/landscaper/pkg/components/model/types"
+	"github.com/gardener/landscaper/pkg/landscaper/registry/componentoverwrites"
+)
+
+type TestComponentVersion struct {
```

Comment:
Why do we need this type? Isn't it the same as the one in .../cnudie/componentversion.go